CONTRIBUTING.txt

```diff
@@ -623,6 +623,31 @@ It is also possible to benchmark features such as peak memory usage. To learn
 more about the features of `asv`, please refer to the official
 `airpseed velocity documentation <https://asv.readthedocs.io/en/latest/writing_benchmarks.html>`_.
 
+Also, the benchmark files need to be importable when benchmarking old versions
+of scikit-image. So if anything from scikit-image is imported at the top level,
+it should be done as:
+
+.. code-block:: python
+
+    try:
+        from skimage import metrics
+    except ImportError:
+        pass
+
+The benchmarks themselves don't need any guarding against missing features,
+only the top-level imports.
+
+To allow tests of newer functions to be marked as "n/a" (not available)
+rather than "failed" for older versions, the setup method itself can raise a
+NotImplemented error.  See the following example for the CWT:
```

Comment:
CWT = continuous wavelet transform? Not sure I understand.

Reply:
Yes, but I didn't meant to leave CWT there.